buildSrc/src/main/kotlin/Dependencies.kt

```diff
@@ -15,28 +15,29 @@
  */
 
 object Versions {
-    const val COMPILE_SDK = 32
+    const val COMPILE_SDK = 34
     const val MIN_SDK = 21
-    const val TARGET_SDK = 32
+    const val TARGET_SDK = 34
 
-    const val AGP = "7.2.2" // https://mvnrepository.com/artifact/com.android.tools.build/gradle?repo=google
-    const val KOTLIN = "1.6.21" // https://mvnrepository.com/artifact/org.jetbrains.kotlin/kotlin-stdlib
-    const val KSP = "1.6.21-1.0.6" // https://mvnrepository.com/artifact/com.google.devtools.ksp/com.google.devtools.ksp.gradle.plugin
-    const val PUBLISH_GRADLE_PLUGIN = "0.21.0" // https://mvnrepository.com/artifact/com.gradle.publish/plugin-publish-plugin?repo=gradle-plugins
+    const val AGP = "8.1.3" // https://maven.google.com/web/index.html#com.android.tools.build:gradle
+    const val KOTLIN = "1.9.20" // https://mvnrepository.com/artifact/org.jetbrains.kotlin/kotlin-stdlib
```

Comment:
Shouldn't the version be 1.9.21 according to the PR's description?

Reply:
It looks like I lost these changes somewhere. Fixed, thanks